deducing-this.md

```diff
@@ -441,7 +530,7 @@ This overload is callable for all `const`- and ref-qualified object parameters,
 
 ## Alternative syntax
 
-Instead of qualifiying the parameter with `this`, to mark it as the parameter for deducing `this`, we could let the first parameter be named `this` instead:
+Instead of qualifying the parameter with `this`, to mark it as the parameter for deducing `this`, we could let the first parameter be *named* `this` instead:
```

Comment:
However, that leaves us at a loss as to how one can address the derived object and the base object, as outlined in the previous section.